content/deyemiobaa/building-a-todolist-with-rails.md

```diff
@@ -0,0 +1,354 @@
+---
+Title: "Bulding a ToDo List with Ruby on Rails"
+Description: "Learn how to build a ToDo List with Ruby on Rails"
+DatePublished: "2022-04-28"
+Categories:
+  - "computer-science"
+  - "web-development"
+Tags:
+  - "Ruby on Rails"
+  - "Ruby"
+  - "PostgreSQL"
+  - "Database"
+CatalogContent:
+  - "learn-ruby"
+  - "learn-rails"
+---
+
+_**Prerequisites:** Ruby, Ruby on Rails, PostgreSQL_
+
+_**Versions:** Ruby 3.2.0, Ruby on Rails 7.0.4, PostgreSQL 15_
+
+## Introduction
+
+This article is an introduction to building a full-stack app using Ruby on Rails. For this article, we are building a To-do list.
+
+I was inspired to write this article while learning Ruby on Rails. I often got lost looking for information on the official Rails guide and I want to help you reduce the time it’ll take you to get started with building your own Rails app.
+
+The article is beginner friendly, and I hope it helps you learn something new.
```

Comment:
[Nitpick] Hmmm... 🤔, there seems to be some ambiguity here. I think it would be beneficial to start your article with the paragraph where you describe your experience. It'll help users relate to you more and make the article less..."technical"(This is the best word I could think of at the moment. Forgive me lol! 😆)?  
  
```suggestion
This article is an introduction to building a full-stack app using Ruby on Rails. For this article, we are building a To-do list.

I was inspired to write this article while learning Ruby on Rails. I often got lost looking for information on the official Rails guide and I want to help you reduce the time it’ll take you to get started with building your own Rails app.

The article is beginner friendly, and I hope it helps you learn something new.
```